frontend/app/controllers/Contributor.scala

```diff
@@ -38,7 +38,7 @@ object Contributor extends Controller with ActivityTracking with PaymentGatewayE
 
   def NonContributorAction = NoCacheAction andThen PlannedOutageProtection andThen authenticated() andThen onlyNonContributorFilter()
 
-  def enterMonthlyContributionsDetails(countryGroup: CountryGroup = UK, contributionValue: Option[Int] = None) = NonContributorAction.async { implicit request =>
+  def enterMonthlyContributionsDetails(countryGroup: CountryGroup = UK, contributionValue: Option[Double] = None) = NonContributorAction.async { implicit request =>
```

Comment:
This the value that receives the template in order to auto-complete the label of the form. Then the `POST` is done using a `BigDecimal` so it is compatible with `membership-common`. If we receive a `BigDecimal` in this method, we have to create Binder just for receiving the value of the label as a query parameter.